src/streamsync/core_ui.py

```diff
@@ -164,9 +164,13 @@ def clear_children(self, component_id: str) -> None:
             try:
                 self.delete_component(child.id)
             except UIError:
+                continue
```

Comment:
We should do it before merging. I would propose something like that :

```python
logger = logging.getLogger("streamsync")
logger.warning("Failed to clear children of component with ID '{component_id}': {child.id} is a builder-managed component.")

Reply:
currently I don't think we have. we would have to design one. I propose the pattern above because it may evolve easily and report logging error in a consistent way till we have better specification on logging functions / module.